test/prism_regression/variables_class.rb

```diff
@@ -0,0 +1,5 @@
+# typed: false
+
+@@class_variable = "value"
```

Comment:
You use `"value"` as your example here, but `1` for global variables below. Literally unintelligible.<sup><sup>(/s)</sup></sup>